### DISCUSSION TITLE
第一次启动时，为什么等待服务器响应的时间这么长？

### DISCUSSION THREAD

**Discussion by wangyu18** [HalseySpicy/Geeker-Admin#151] [Q&A]
本地启动的。
app.vue和main.ts都是这样，等待服务器响应的时间为什么这么长？是在等待什么呢?
求解答

![20230315151348](https://user-images.githubusercontent.com/32888969/225234591-c2476326-609c-4231-9b09-517224429d2b.png)

**Comment by HalseySpicy** [comment]
![image](https://user-images.githubusercontent.com/51069636/225262921-f9c1c0f3-b3f4-4bd5-8298-7e9ac50cc6fc.png)

**Comment by HalseySpicy** [comment]
意思大概就是，第一次启动 vite 服务，虽然说启动了，但是需要编译 node_modules 下面的所有包依赖，编译完成会在本地形成缓存，后面启动都很快了
